averages.js

```diff
@@ -0,0 +1,30 @@
+
+var total = Array.prototype.total = function() {
+  var array = this;
+  array.reduce(function (a, b) {
+    return a + b;
+  }, 0)
+}
+
+var mean = Array.prototype.mean = function() {
+  var array = this;
+  length = array.length
+  sum = array.reduce(function (a, b) {
+    return a + b;
+  }, 0)
+  return sum / length
+}
+
+var median = Array.prototype.median = function() {
```

Comment:
i take this back!!! the challenge asks you to do this (whooooops on my part!) - this is great 👍